addons/web/static/src/core/errors/error_dialogs.js

```diff
@@ -33,6 +33,7 @@ export const odooExceptionTitleMap = new Map(
         "odoo.exceptions.AccessDenied": _t("Access Denied"),
         "odoo.exceptions.MissingError": _t("Missing Record"),
         "odoo.addons.web.controllers.action.MissingActionError": _t("Missing Action"),
+        "odoo.addons.base.models.ir_actions.ServerActionWithWarningsError": _t("Server Action With Warnings"),
```

Comment:
```suggestion
        "odoo.addons.base.models.ir_actions.ServerActionWithWarningsError": _t("Invalid Operation"),
```

- `Server Action With Warnings` seems off in the warning modal IMO.
- More importantly, it means we would benefit from that term already translation-exported (.pot file) and translated (transifex), seems convenient and killing two birds with one stone

<img width="1208" height="216" alt="image" src="https://github.com/user-attachments/assets/c091b411-220a-4a8d-8361-e0c734a42ae9" />

If you don't agree and want to keep it, it's fine, framework-js will say what they want :+1: